src/Aspire.Hosting.AWS/Lambda/LambdaEmulatorAnnotation.cs

```diff
@@ -14,4 +14,22 @@ internal class LambdaEmulatorAnnotation(EndpointReference endpoint) : IResourceA
     /// The HTTP endpoint for the Lambda runtime emulator.
     /// </summary>
     public EndpointReference Endpoint { get; init; } = endpoint;
+
+    /// <summary>
+    /// If set to true Amazon.Lambda.TestTool will updated/installed during AppHost startup. Amazon.Lambda.TestTool is 
```

Comment:
the name and the description are saying 2 different things. The name suggests that we won't auto install, but the description is saying that we will?

Reply:
Fixed